urlimport.go

```diff
@@ -0,0 +1,201 @@
+package main
+
+import "fmt"
+import "flag"
+import "net/http"
+import "os"
+import "io"
+import "io/ioutil"
+import "net/url"
+import "strings"
+import "time"
+import "net"
+
+import "github.com/jlaffaye/ftp"
```

Comment:
This is a minor thing, but most Go code that I've seen groups all of the imports in a single import statement:

```
import (
    "fmt"
    "flag"
    "net/http"
    "os"
    "io"
    "io/ioutil"
    "net/url"
    "strings"
    "time"
    "net"

    "github.com/jlaffaye/ftp"
)
```